package.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "@flowforge/localfs",
+    "name": "@flowfuse/driver-localfs",
```

Comment:
This will need changes to the `forge/container/index.js` code as we use the driver name as the key to load

Also in `forge/config/index.js` where we default to `localfs`

Reply:
This is dealt with in https://github.com/FlowFuse/flowfuse/pull/3141